src/Kentico.Xperience.Lucene.Admin/Client/src/templates/LuceneIndexConfigurationFormComponent.tsx

```diff
@@ -1,451 +1,486 @@
 import { type FormComponentProps } from '@kentico/xperience-admin-base';
 import {
-    type ActionCell,
-    Button,
-    ButtonType,
-    CellType,
-    ColumnContentType,
-    Input,
-    Stack,
-    type StringCell,
-    Table,
-    type TableAction,
-    type TableCell,
-    type TableColumn,
-    type TableRow,
+  type ActionCell,
```

Comment:
No significant functionality changes were made to this file. This is all eslint and formatting by prettier (which is one of the Client project dependencies).